jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3827,7 +3827,51 @@ static String valueOfCodePoint(int codePoint) {
 		}
 		return string;
 	}
-	/*[ENDIF] Java11*/
+
+
+	/**
+	 * Returns a string whose value is the concatenation of this string repeated
+	 * count times. 
+	 * 
+	 * @param count
+	 *            a positive integer indicating the number of times to be repeated
+	 * @return a string whose value is the concatenation of this string repeated count times
+	 * @throws IllegalArgumentException
+	 *             if the count is negative
+	 * @since 11
+	 */
+	public String repeat(int count) {
+		if (count < 0) {
```

Comment:
what happens if the count is zero

Reply:
If the count is zero then a byte array of length 0 is passed into the String constructor and an empty String is returned as desired. Should this be done more explicitly?

Reply:
Ya, I think it would be beneficial to provide a faster path for `count ==0` as well as `this.equals("")` just like you have done for `count==1`